auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Routing.kt

```diff
@@ -99,7 +134,13 @@ fun Route.deltaLoginRoutes(
     }
 }
 
+fun deltaRouteWithEmail(deltaUrl: String, ssoClientInternalId: String, email: String) =
+    deltaUrl + "/oauth2/authorization/delta-auth?sso-client=${ssoClientInternalId}&expected-email=${email.encodeURLParameter()}"
+
 fun oauthClientLoginRoute(ssoClientInternalId: String) = "/delta/oauth/${ssoClientInternalId}/login"
+fun oauthClientLoginRouteWithEmail(ssoClientInternalId: String, email: String) =
+    "/delta/oauth/${ssoClientInternalId}/login?expected-email=${email.encodeURLParameter()}"
```

Comment:
Could make this one function with a default parameter i.e. `oauthClientLoginRouteWithEmail(ssoClientInternalId: String, email: String? = null)` (not saying that would be better, just making sure you know it's an option!)